configure.ac

```diff
@@ -500,7 +500,7 @@ AS_IF([test x"$with_cuda" != x"no"],
 			      [])],
 	    [])
 
-AS_IF([test "$with_cuda" = "yes" && test "$have_libcuda" = "0" ],
```

Comment:
Can we skip the `test -n` by setting `with_cuda = no` in the default case?

Reply:
That was my original PR last night, but then I remembered the discussion in https://github.com/ofiwg/libfabric/pull/6294 where Sean preferred to have the package check happen by default (when with_cuda is not specified at all).

Reply:
Ah, looks good to me then.